src/Goerli-DssSpell.sol

```diff
@@ -16,26 +16,51 @@
 
 pragma solidity 0.6.12;
 // Enable ABIEncoderV2 when onboarding collateral through `DssExecLib.addNewCollateral()`
-pragma experimental ABIEncoderV2;
+// pragma experimental ABIEncoderV2;
 
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
 
 import { DssSpellCollateralAction } from "./Goerli-DssSpellCollateral.sol";
 
-interface StarknetBridgeLike {
-    function close() external;
-    function isOpen() external returns (uint256);
+interface TeleportJoinLike {
+    function rely(address) external;
+    function file(bytes32,address) external;
+    function file(bytes32,bytes32,address) external;
+    function file(bytes32,bytes32,uint256) external;
+    function vat() external view returns (address);
+    function daiJoin() external view returns (address);
+    function ilk() external view returns (bytes32);
+    function domain() external view returns (bytes32);
 }
```

Comment:
```suggestion
interface TeleportJoinLike {
    function file(bytes32,bytes32,address) external;
    function file(bytes32,bytes32,uint256) external;
}
```

Reply:
The full interfaces was intentionally left so it would align with arbitrum and optimism onboardings.
Everywhere it was possible I opted for easy diffing and leaving the same things.
Let me know if you still think it should be changed.

Reply:
right, I'm taking the suggestions.